nginx/nginx.conf

```diff
@@ -17,6 +17,8 @@ http {
 	'$status $body_bytes_sent "$http_referer" '
 	'"$http_user_agent" "$http_x_forwarded_for"';
 
+	access_log /var/log/nginx/access.log main;
```

Comment:
We may also use `/dev/stdout` to be more explicit as this is already what `nginx` does:

https://github.com/nginxinc/docker-nginx/blob/aa41ddeef871b7f0ea64a44f26d3f4aa0e6d5e7b/mainline/buster/Dockerfile#L95-L97

Reply:
Using the symlinks that the Nginx Docker image already provides means that anyone who already uses the Docker Nginx container will be familiar with what is going on.

This way, if someone wants to redirect the nginx logs to a file on disk, all they need to do is add a `volume` entry to the `docker-compose.yml` file for the `/var/log/nginx` directory (or for the `access.log` and/or `error.log` files). However, if we explicitly default to logging to `/dev/stdout`, then someone who wants to make that change will have to change this file too and might not expect to need to do that.

I think that sticking with the Nginx default log paths is cleaner and easier to use.

Reply:
Great reasoning, thanks a lot for taking the time for this detailed response!